### PR TITLE
Always enable manager

### PR DESCRIPTION
**Pull request by filips**

### CODE REVIEW

File: charts/codesealer/values.yaml [hunk 1]
```diff
@@ -107,6 +107,9 @@ webhook:
 
   # affinity: {}
 
+sidecar:
```
Comment (psve) [inline charts/codesealer/values.yaml:110]: We should document this with a comment so it shows up in `helm show values`.

File: charts/codesealer/values.yaml [hunk 2]
```diff
@@ -107,6 +107,9 @@ webhook:
...
+  enabled: false
```
Comment (psve) [inline charts/codesealer/values.yaml:112]: Can't comment on that line, but on line 606 there's still a `manager.enabled = false`. I suppose that should be removed?

File: charts/codesealer/README.md [hunk 3]
```diff
@@ -158,6 +159,8 @@ helm install codesealer ${CODESEALER_HELM_CHART} \
   --set worker.ingress.nodePort="${INGRESS_NODEPORT}" \
   --set worker.redis.namespace="${REDIS_NAMESPACE}" \
   --set worker.config.bootloader.redisPassword="${REDIS_PASSWORD}" \
+  --set sidecar.enabled=true \
+  --set environment=${CODESEALER_ENV} \
```
Comment (psve) [inline charts/codesealer/README.md]: What is this? I don't see a mention of `CODESEALER_ENV` anywhere else?

File: charts/codesealer/README.md [hunk 4]
```diff
@@ -183,43 +186,13 @@ To see what Codesealer helm parameters are available issue the following command
...
->
-> ```bash
->kubectl port-forward service/core-manager -n ${INGRESS_NAMESPACE} 84444:8444 &
-> ```
-> You can access the manager at https://localhost:8444
->
+To access local manager issue the following command:
+
+ ```bash
+kubectl port-forward service/core-manager -n ${INGRESS_NAMESPACE} 84444:8444 &
```
Comment (psve) [inline charts/codesealer/README.md]: Two things:
* The manager is deployed in the `codesealer-system` namespace,
* We should mention how to get the admin password.
Basically, just copy what's in the 'NOTES.txt` about this.

File: charts/codesealer/README.md (hunk 3 above)
Comment (filips) [reply charts/codesealer/README.md]: Not really sure, suppose it came in through a rebase. Removed

**Review by psve — APPROVED** [review]
Seems to work :+1:

**Pull request merged**